src/shared/sentry/sentryadapter.cpp

```diff
@@ -159,6 +171,7 @@ void SentryAdapter::declineCrashReporting() {
 }
 
 void SentryAdapter::captureQMLStacktrace(const char* description) {
+#ifndef UNIT_TEST
```

Comment:
Is this because we don't want to report crashes for unit tests?

Reply:
No sadly this is me having a problem getting this to build on github CI. 
For an (for me) unknown reason, the compiler cannot find the private headers i.e  `include <QtQml/private/qqmlengine_p.h>` - which will cause this to fail if we build the tests. 
Given that we currently have no callers when running tests, i'd like to disable this and fix it once we actually need that.